components/frontapp/sources/common/polling-ids.mjs

```diff
@@ -0,0 +1,55 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import frontapp from "../../frontapp.app.mjs";
+
+export default {
+  props: {
+    frontapp,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+  },
+  methods: {
+    _getSavedIds() {
+      return this.db.get("savedIds") || [];
+    },
+    _setSavedIds(ids) {
+      this.db.set("savedIds", ids);
+    },
```

Comment:
Would it be simpler to use a timestamp to check the events, rather than storing all the ID's?

If that's not an option, storing the ID's in an object (e.g. { 123: true, 456: true }) would reduce execution time. Checking whether savedIds[id] is `true` is faster than using savedIds.includes().

Reply:
Thanks for the suggeston, I've changed the two sources to adhere to the timestamp format, with a few modifications to the base file to support them